docs/release/trg-5/trg-5-07.md

```diff
@@ -50,4 +50,4 @@ The following table describes the required version of dependencies for all produ
 
 | Name       | Repo URL                                      | Chart version | App version |
 |------------|-----------------------------------------------|:-------------:|:-----------:|
-| postgresql | [Bitnami](https://charts.bitnami.com/bitnami) |    12.x.x     |   15.x.x    |
+| postgresql | [Bitnami](https://charts.bitnami.com/bitnami) | >=15.4 <16    | >=16.3 <17  |
```

Comment:
```suggestion
| postgresql | [Bitnami](https://charts.bitnami.com/bitnami) | `>=15.4 <16` | `>=16.3 <17` |
```

Reply:
because of the mdx checker >, < needs to be escaped. Should solve the website build issue